packages/app/src/cli/commands/app/function/run.ts

```diff
@@ -78,12 +78,18 @@ export default class FunctionRun extends Command {
           )
         }
 
+        const inputQueryPath = ourFunction?.configuration.targeting?.[0]?.input_query
+        const queryPath = inputQueryPath && `${ourFunction?.directory}/${inputQueryPath}`
+        const schemaPath = await readSchemaFile(ourFunction?.directory || '')
```

Comment:
If there isn't a schema, should we run the `generate schema` command first automatically?

Reply:
Yea I think that makes sense - will update!

Reply:
Updated this to call the generate schema.  It will output the message from the generate schema when it has been created:

<img width="1164" alt="Screenshot 2024-09-11 at 4 28 26 PM" src="https://github.com/user-attachments/assets/69854687-25d2-493b-b585-c00ccf19598b">